chan-plugin/chan-plugin.php

```diff
@@ -0,0 +1,85 @@
+<?php /**
+ * Plugin Name: My Simple Plugin
+ * Plugin URI: ''
+ * Author: Chandan
+ * Author URI: ''
+ * Version: 2.3.5
+ * Description: A new Plugin
+ * Text Domain: 'My-Simple-Plugin'
+ * *
+ * 
+ */
+
+ 
+ /**
+  * Handles a shortcode on posts
+  *
+  * @since 8.1.9
+  *
+  * @return HTML snippet which includes shortcode default attributes if user haven't provided any attributes.
+  */
+
+ function handle_shortcode( $attr, $content='' ){
+    //global post variable
+    global $post;       
+
+    //default attribute values
+    $attr = shortcode_atts( array(      
+        'title' => __('Default title', 'My-Simple-Plugin'),
+        'color' => __('red', 'My-Simple-Plugin'), 
```

Comment:
**red** don't need to be translated
user would like to translate strings only which appears on frontend

Reply:
Translation has been removed for color key